reports/guild-count.js

```diff
@@ -0,0 +1,20 @@
+// How many Guilds are users members of? Outputs counts of users with each number of Guilds
+
+var aggregationCursor = db.users.aggregate(
```

Comment:
Maybe we should just add this directory to linting? Since I think we will be adding on to this.